server/views/partials/email-templates/health-questions.njk

```diff
@@ -1,8 +1,12 @@
-<ol>
-  <li>Do they have any physical health needs? Please describe them and the level of severity.</li>
-  <li>Are they currently receiving any medical treatment for their physical health needs? If so, please describe it.</li>
-  <li>Are they currently taking any medication for their physical health needs? If so, please provide details.</li>
-  <li>Can they climb stairs?</li>
-  <li>Can they live independently?</li>
-  <li>What support is required?</li>
-</ol>
+<div id="health-questions-template">
+  <ol>
+    <li>Do they have any physical health needs? Please describe them and the level of severity.</li>
+    <li>Are they currently receiving any medical treatment for their physical health needs? If so, please describe it.</li>
+    <li>Are they currently taking any medication for their physical health needs? If so, please provide details.</li>
+    <li>Can they climb stairs?</li>
+    <li>Can they live independently?</li>
+    <li>What support is required?</li>
+
+    <button id="health-questions-button">Copy text</button>
```

Comment:
copy functionality seems to work great 🥳 🙌 
but because the `Copy text` button is within the same `div` the button text gets copied to. Could the button be moved to a div outside the `drug-alcohol-questions-template` one (and same for other templates)